### PR TITLE
Feat/live 2804/headers andgraphs

### PR DESCRIPTION
**Pull request by mlegall-ledger**
### 📝 Description

Created new Header Layout component to avoid animations duplication
Modified GraphCards contents for both Account and Asset pages
Modified general animations.

### ❓ Context

- **Impacted projects**: `` 
- **Linked resource(s)**: `` 

### ✅ Checklist

- [ ] **Test coverage** 
- [ ] **Atomic delivery** 
- [ ] **No breaking changes** 

### 📸 Demo

https://user-images.githubusercontent.com/39890664/185371622-3e1068f8-f30c-4197-bffb-a70006fbe2a8.mov

### 🚀 Expectations to reach

_Please make sure you follow these [**Important Steps**](https://github.com/LedgerHQ/ledger-live/blob/develop/CONTRIBUTING.md#important-steps)._

_Pull Requests must pass the CI and be internally validated in order to be merged._

### CODE REVIEW

File: apps/ledger-live-mobile/src/screens/Account/AccountHeader.tsx [hunk 1]
```diff
@@ -52,95 +72,47 @@ function AccountHeader({
...
+        <Flex flexDirection={"column"} alignItems={"center"}>
+          {typeof items[1]?.value === "number" && account?.name ? (
+            <>
+              <Text
+                variant={"small"}
+                fontWeight={"semiBold"}
+                color={"neutral.c70"}
+                fontSize="11px"
+              >
+                {account.name}
```
Comment (cgrellard-ledger) [inline apps/ledger-live-mobile/src/screens/Account/AccountHeader.tsx:94]: If we are on a token account we should display `parentAccount.name - account.name`

File: apps/ledger-live-mobile/src/components/AssetCentricGraphCard.tsx [hunk 2]
```diff
@@ -40,8 +40,8 @@ import getWindowDimensions from "../logic/getWindowDimensions";
 import Graph from "./Graph";
 import Touchable from "./Touchable";
 import TransactionsPendingConfirmationWarning from "./TransactionsPendingConfirmationWarning";
-import { NoCountervaluePlaceholder } from "./CounterValue";
-import DiscreetModeButton from "./DiscreetModeButton";
+import ParentCurrencyIcon from "./ParentCurrencyIcon";
+import FormatDate from "./FormatDate";
```
Comment (cgrellard-ledger) [inline apps/ledger-live-mobile/src/components/AssetCentricGraphCard.tsx:45]: missing declaration for BigPlaceholder

File: apps/ledger-live-mobile/src/components/CurrencyHeaderLayout.tsx [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+function CurrencyHeaderLayout({
+  currentPositionY,
+  graphCardEndPosition,
+  rightElement,
+  centerAfterScrollElement,
+  centerBeforeScrollElement,
+  leftElement,
+  currencyColor,
+}: {
+  currentPositionY: SharedValue<number>;
```
Comment (nparigi-ledger) [inline apps/ledger-live-mobile/src/components/CurrencyHeaderLayout.tsx:24]: Import SharedValue

File: apps/ledger-live-mobile/src/screens/Account/AccountHeader.tsx [hunk 4]
```diff
@@ -1,47 +1,67 @@
...
 function AccountHeader({
   currentPositionY,
   graphCardEndPosition,
   account,
+  useCounterValue,
+  counterValueCurrency,
+  history,
+  countervalueAvailable,
 }: {
   currentPositionY: SharedValue<number>;
```
Comment (nparigi-ledger) [inline apps/ledger-live-mobile/src/screens/Account/AccountHeader.tsx:33]: import SharedValue

File: apps/ledger-live-mobile/src/screens/Account/AccountHeader.tsx [hunk 5]
```diff
@@ -52,95 +72,47 @@ function AccountHeader({
...
+      }
+      centerAfterScrollElement={
+        <Flex flexDirection={"column"} alignItems={"center"}>
+          {typeof items[1]?.value === "number" && account?.name ? (
+            <>
+              <Text
+                variant={"small"}
+                fontWeight={"semiBold"}
+                color={"neutral.c70"}
+                fontSize="11px"
```
Comment (nparigi-ledger) [inline apps/ledger-live-mobile/src/screens/Account/AccountHeader.tsx:92]: fontSize="11px" BAD

File: apps/ledger-live-mobile/src/screens/Account/AccountHeader.tsx [hunk 6]
```diff
@@ -52,95 +72,47 @@ function AccountHeader({
...
+              >
+                {account.name}
+              </Text>
+              <Text
+                variant={"small"}
+                fontWeight={"semiBold"}
+                color={"neutral.c100"}
+                fontSize="18px"
```
Comment (nparigi-ledger) [inline apps/ledger-live-mobile/src/screens/Account/AccountHeader.tsx:100]: fontSize="18px" VERY BAD
It's fontSizes.h3 (with Inter maybe)

File: apps/ledger-live-mobile/src/screens/Account/AccountHeader.tsx (hunk 5 above)
Comment (nparigi-ledger) [reply apps/ledger-live-mobile/src/screens/Account/AccountHeader.tsx:92]: It's fontSizes.subtitle

File: apps/ledger-live-mobile/src/screens/WalletCentricAsset/Header.tsx [hunk 7]
```diff
@@ -47,97 +53,62 @@ function Header({
...
+      }
+      centerAfterScrollElement={
+        <Flex flexDirection={"column"} alignItems={"center"}>
+          {balanceAvailable ? (
+            <>
+              <Text
+                variant={"small"}
+                fontWeight={"semiBold"}
+                color={"neutral.c70"}
+                fontSize="11px"
```
Comment (nparigi-ledger) [inline apps/ledger-live-mobile/src/screens/WalletCentricAsset/Header.tsx:80]: variant=subtitle

File: apps/ledger-live-mobile/src/screens/WalletCentricAsset/Header.tsx [hunk 8]
```diff
@@ -47,97 +53,62 @@ function Header({
...
+              >
+                {t("asset.title", { assetName: currency.name })}
+              </Text>
+              <Text
+                variant={"small"}
+                fontWeight={"semiBold"}
+                color={"neutral.c100"}
+                fontSize="18px"
```
Comment (nparigi-ledger) [inline apps/ledger-live-mobile/src/screens/WalletCentricAsset/Header.tsx:88]: variant h3

**Pull request merged**